addons/payment/static/src/js/payment_form_mixin.js

```diff
@@ -231,12 +244,14 @@ odoo.define('payment.payment_form_mixin', require => {
         _hideError: () => this.$('div[name="o_payment_error"]').remove(),
```

Comment:
Shouldn't `_hideError` be a function too?

Reply:
Indeed, it should. I didn't bother because it wasn't a big deal, but it'll be weird to see an error on a first form disappear when you get another error on a second form.